example/package.json

```diff
@@ -6,12 +6,17 @@
     "android": "react-native run-android",
     "ios": "react-native run-ios",
     "start": "react-native start",
-    "test": "jest"
```

Comment:
None of this will persist. I have no intention of maintaining a static example and taking on the pointless work of keeping it up to date with react-native versions, it is clobbered every time by the refresh-example.sh script